src/blind.cpp

```diff
@@ -255,8 +252,14 @@ int BlindTransaction(std::vector<uint256 >& input_blinding_factors, const std::v
 
     // Needed to construct the proof itself. Generators must match final transaction to be valid
     std::vector<secp256k1_generator> targetAssetGenerators;
-    surjectionTargets.resize(tx.vin.size()*3);
-    targetAssetGenerators.resize(tx.vin.size()*3);
+
+    size_t maxTargets = tx.vin.size()*3;
```

Comment:
Maybe note here that this is a strict upper-bound, and the size is likely less(and is shrunk later).

Reply:
added the comment